cmd/nerdctl/run_linux_test.go

```diff
@@ -331,3 +332,14 @@ func TestRunWithFluentdLogDriverWithLogOpt(t *testing.T) {
 	assert.Equal(t, true, strings.Contains(logData, "test2"))
 	assert.Equal(t, true, strings.Contains(logData, inspectedContainer.ID))
 }
+
+func TestRunWithOOMScoreAdj(t *testing.T) {
```

Comment:
I think it works well on docker because I have docker added in the sudo'ers group.
Code here shows rootless mode cannot be supported: https://github.com/moby/moby/blob/cf867587b9dc8edf92efd8372fcc6f86f117e386/daemon/daemon_unix.go#L1642-L1656